### PR TITLE
misc patches

### PR DESCRIPTION
**Pull request by spiralw**
- feat: case-insensitive "text" keyword matching for proxy tags
- fix: don't try matching commands if there is nothing to match
- feat: show member color in reply embed

### CODE REVIEW

File: PluralKit.Bot/Commands/MemberProxy.cs [hunk 1]
```diff
@@ -26,6 +26,7 @@ ProxyTag ParseProxyTags(string exampleProxy)
             {
                 // // Make sure there's one and only one instance of "text" in the example proxy given
                 var prefixAndSuffix = exampleProxy.Split("text");
+                if (prefixAndSuffix.Length == 1) prefixAndSuffix = prefixAndSuffix[0].Split("TEXT");
```
Comment (xSke) [inline PluralKit.Bot/Commands/MemberProxy.cs:29]: hmm, is there a way to do this "nicer"? like a case-insensitive split?

File: PluralKit.Bot/Proxy/ProxyService.cs [hunk 2]
```diff
@@ -194,11 +194,14 @@ private Embed CreateReplyEmbed(Message trigger, Message repliedTo, string? nickn
             var username = nickname ?? repliedTo.Author.Username;
             var avatarUrl = $"https://cdn.discordapp.com/avatars/{repliedTo.Author.Id}/{repliedTo.Author.Avatar}.png";
 
+            Console.WriteLine($"color {match.Member.Color}");
```
Comment (xSke) [inline PluralKit.Bot/Proxy/ProxyService.cs:197]: remove this

File: PluralKit.Bot/Commands/MemberProxy.cs (hunk 1 above)
Comment (spiralw) [reply PluralKit.Bot/Commands/MemberProxy.cs:29]: could use a regex instead, but that seems a bit overkill?

Comment (xSke) [reply PluralKit.Bot/Commands/MemberProxy.cs:29]: yeah, fair.

**Pull request merged**